pkg/app/piped/executor/ecs/deploy.go

```diff
@@ -17,6 +17,7 @@ package ecs
 import (
 	"context"
 
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
```

Comment:
```suggestion
	"github.com/aws/aws-sdk-go-v2/service/ecs/types"

```
go import convention

Reply:
thanks, I fixed it.